packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/listGrid.html.twig

```diff
@@ -14,4 +14,4 @@
 {% block grid_pager_totalcount %}
     {% set entityName = 'category'|trans %}
     {{ parent() }}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
do you know where is this template used? I did not find the usage...

Reply:
that is a very good question :smile: even the content is somewhat weird, I removed it in 88aba0c18